src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -11,7 +11,7 @@
 namespace Microsoft.ML
 {
     /// <summary>
-    /// Specifies input and output column names for a transformation.
+    /// Specifies input and output column names for transformer components that operate on multiple input/output columns.
```

Comment:
columns.  It is often advantageous to transform several columns at once as all of the changes can be done in a single data pass.